triton/stream.py

```diff
@@ -262,15 +272,27 @@ def _put_many_packed(self, records):
         resp_value = []
         num_records = len(records)
         max_record = 0
+        retry_records = []
         while max_record < num_records:
-            resp = self.conn.put_records(
+            resp = _call_and_retry(
```

Comment:
I don't understand why `call_and_retry` is used here and there is still a sleep below.